### ISSUE TITLE
Dragging custom element into multi column element: Element type is invalid

### ISSUE THREAD

**Issue by kishortrimal** [Kiho/react-form-builder#131]
![image](https://user-images.githubusercontent.com/21039746/114706183-4a044f00-9d46-11eb-8f71-c9a27bc19bed.png)

**Check my code:**

```
import React, { useState, useEffect} from 'react';
import { ReactFormBuilder, ElementStore, Registry } from 'react-form-builder2';
import { Form, Button, Input, Card } from 'antd'
import 'react-form-builder2/dist/app.css';

const form_builder = () => {

  const TestComponent = () => <h2>Hello</h2>;

  const [flag, setFlag] = useState(false);

  const AntDesignInput = React.forwardRef((props, ref) => {
    const { name, defaultValue, disabled } = props;
    return <Input ref={ref} name={name} defaultValue={defaultValue} disabled={disabled} />;
  });
  console.log(AntDesignInput);

  useEffect(() => {
    Registry.register('AntInput', AntDesignInput);
    setFlag(true);
  }, []);

   const items = [{
      key: 'Header',
    }, {
      key: 'TextInput',
    }, {
      key: 'TextArea',
    }, {
      key: 'RadioButtons',
    }, {
      key: 'Checkboxes',
    }, {
      key: 'Image',
    },
    {
      key: 'TwoColumnRow'
    },
    {
      key: 'ThreeColumnRow'
    },
    {
      key: 'FourColumnRow'
    },
    {
      key: 'TestComponent',
      element: 'CustomElement',
      component: TestComponent,
      type: 'custom',
      field_name: 'test_component',
      name: 'Something You Want',
      icon: 'fa fa-cog',
      static: true,
      props: { test: 'test_comp' },
      label: 'Label Test',
    },
    {
      key: 'AntInput',
      element: 'CustomElement',
      component: AntDesignInput,
      forwardRef: true,
      type: 'custom',
      field_name: 'my_input_',
      name: 'Ant Design Input',
      icon: 'fa fa-cog',
      label: 'Label Input',
    }
   ];

   return(
     (flag) && (
      <ReactFormBuilder toolbarItems={items}/>
     )

   )
}
export default form_builder;

```
In above code, I have used custom element and custom element working properly but when, I am trying to drag my custom element into multi column row element. getting above error.

check snapshot of UI screens:

![image](https://user-images.githubusercontent.com/21039746/114709211-eda32e80-9d49-11eb-9d5d-5ebb969dc3c1.png)
 
![image](https://user-images.githubusercontent.com/21039746/114709351-19beaf80-9d4a-11eb-947c-f32333790daf.png)

![image](https://user-images.githubusercontent.com/21039746/114710608-8d14f100-9d4b-11eb-8a67-0be0bd80902b.png)

![image](https://user-images.githubusercontent.com/21039746/114709770-9c476f00-9d4a-11eb-8114-cdfe03e3a8da.png)

**Comment by kishortrimal** [comment]
Hello kiho,
 Drag issue is solved but when, I drag custom element not able to edit or delete custom element in multi element row. please refer below snap short.

![image](https://user-images.githubusercontent.com/21039746/114846624-809ba180-9dfa-11eb-9176-ded25c0416de.png)

**Comment by Kiho** [comment]
It should be fixed in v 0.8.5
